src/Domain/Structure/Neighbors.hpp

```diff
@@ -31,8 +31,22 @@ class Neighbors {
                 std::is_same_v<IdType, ElementId<VolumeDim>>);
 
  public:
+  /// Construct with the ids, the orientation of the neighbors relative to the
+  /// host, and whether the neighbors are conforming.
+  ///
+  /// \param ids the ids of the neighbors.
+  /// \param orientations An OrientationMap (which takes objects in the logical
+  /// coordinate frame of the host and maps them to the logical coordinate frame
+  /// of the neighbor) for each neighboring Block (Elements within a Block share
+  /// the same orientation).
```

Comment:
Is the key in this may the block ID or the `IdType`?

Reply:
I'll state that it is a size_t for the block ID